lib/backends/secrets-manager-backend.js

```diff
@@ -41,8 +42,15 @@ class SecretsManagerBackend extends KVBackend {
       })
     }
 
+    let params
+    if (versionId == null) {
+      params = { SecretId: key, VersionStage: versionStage }
+    } else {
+      params = { SecretId: key, VersionId: versionId }
+    }
```

Comment:
Lets flip the if and just do a truthy check. Otherwise I want to add another `=` so its a strict check `===` , yay for JS. But 

```suggestion
    if (versionId) {
      params = { SecretId: key, VersionId: versionId }
    } else {
      params = { SecretId: key, VersionStage: versionStage } 
    }
```